R/SS_read.R

```diff
@@ -5,29 +5,28 @@
 #' Functionality comes from the `r4ss::SS_read*()` functions.
 #' This function simplifies the number of lines of code you need to write by
 #' using all of the read functions to read in the
-#' starter, control, data, and forecast files.
+#' starter, control, data, and forecast files and maybe the weight-at-age file.
 #' The starter file is helpful because it provides names for the
 #' control and data files.
 #'
-#' @template dir
+#' @param dir A file path to the directory of interest or a raw github URL (see
+#' example). The default value is \code{dir = NULL}.
 #' @param ss_new A logical that controls if the `.ss_new` files or
 #'   the original input files are read in.
 #'   The default is to read the original files.
 #' @template verbose
-#'
 #' @author Ian G. Taylor, Kelli F. Johnson
-#'
 #' @return An invisible list is returned.
 #' The first element is the directory that was provided in the argument `dir`.
 #' The second element is the result of `normalizePath(dir)`,
 #' which gives the full path.
-#' The remaining four elements are list objects from reading in
+#' The remaining five elements are list objects resulting from reading in
 #' the following input files:
 #' * data
 #' * control
 #' * starter
 #' * forecast
-#' * wtatage (will be NULL if not required by the model)
+#' * wtatage (will be `NULL` if not required by the model)
```

Comment:
If a list object is `NULL`, it is often just dropped from the object. If there should defintely be a list object called wtatage even if wtatage is read in, it should be assigned `NA` instead.

Reply:
I think it is fine as of right now to leave this as NULL but maybe I can take a stab at changing the documentation to reflect what you said b/c knowing that behavior is helpful, or at least it was for me.

Reply:
I re-read the documentation and I think it is fine as is.

Reply:
Ok!